news.md

```diff
@@ -3,6 +3,16 @@ title: News
 ---
 ## News
 
+### Jython 2.7.2 release (v2.7.2 21 March 2020)
+
+A new full release is now available for Jython 2.7.2 at
+[Maven Central](https://search.maven.org/search?q=g:org.python).
+It is built and tested with Java 8 and tested against Java 11.
+There are no changes (apart from the version identification) from v2.7.2rc1.
+
+Fixed bugs for all versions are listed in [NEWS](https://github.com/jythontools/jython/blob/v2.7.2/NEWS).
+
+
 ### Jython 2.7.2 release candidate (v2.7.2rc1 March 2020)
```

Comment:
It was March 5 if you want to retrofit this date

Reply:
Yeah, why not.